ruleset/sca/amazon/cis_amazon_linux_2.yml

```diff
@@ -875,10 +844,10 @@ checks:
       - tsc: ["CC5.2"]
     condition: none
     rules:
-      - 'c:systemctl is-enabled avahi-daemon -> r:enabled'
+      - 'p: avahi-daemon'
```

Comment:
We have to check if Avahi is installed, not running
```
# rpm -q avahi-autoipd avahi
```

So the rules should look like:
```yml
condition: all
rules:
  - 'c:rpm -q avahi-autoipd -> r:package avahi-autoipd is not installed'
  - 'c:rpm -q avahi -> r:package avahi is not installed'
```